client/my-sites/plan-features/header.jsx

```diff
@@ -51,13 +51,15 @@ class PlanFeaturesHeader extends Component {
 	}
 
 	renderPlansHeader() {
-		const { planType, popular, newPlan, title, translate } = this.props;
+		const { newPlan, planType, popular, selectedPlan, title, translate } = this.props;
 
 		const headerClasses = classNames( 'plan-features__header', getPlanClass( planType ) );
+
 		return (
 			<header className={ headerClasses } onClick={ this.props.onClick }>
-				{ popular && <Ribbon>{ translate( 'Popular' ) }</Ribbon> }
-				{ newPlan && <Ribbon>{ translate( 'New' ) }</Ribbon> }
+				{ planType === selectedPlan && <Ribbon>{ translate( 'Suggested' ) }</Ribbon> }
```

Comment:
Note: this will not work for the corresponding plan from the other plan period (so if we have `?plan=jetpack_premium_monthly`, it won't work for JP yearly premium plan). Is that intentional?

Reply:
This is true. Current landing view is `yearly`, so that nudges pass the yearly constant. 
That said, let's play it safe as in https://github.com/Automattic/wp-calypso/issues/20195. 

Note: same should happen for https://github.com/Automattic/wp-calypso/pull/20204